varcode/effects/effect_ordering.py

```diff
@@ -125,13 +124,31 @@ def effect_sort_key(effect):
     effect_class_priority = effect_priority(effect)
     return (effect_class_priority, cds_length, transcript_length)
 
+
+def select_between_exonic_splice_site_and_alternate_effect(effect):
```

Comment:
can you add some comments here? this functionality is kinda non-obvious

Reply:
Added a docstring